codelabs/codelab-03/README.md

```diff
@@ -0,0 +1,143 @@
+
+Your form should look similar to this.
+
+![public_read_access Page](../../../media/codelabs/codelab-03/public_read_access.png)
+
+Click the `Upload` button in the lower-left corner.
+
+#### Testing latency
+
+Select your new object by again hovering over the name until highlighted with an underscore and clicking. This page similarly lets lets you see some details and configure properties. What we're after is the URL at the bottom of the page, under the label `Link`.
+
+Copy the link and paste it in the URL bar of a new tab in your browser. The canyon should start loading, albeit slowly. Why so slow? Well, we did place it in a region half-way 'round the world. 
+
+Lets try to quantify that speed. Edit the last line in the included shell script `lat_test.sh`, using a command line text editor or IDE, and drop in the copied link. If you haven't already, enter your environment now by running:
+
+```
+ `$pipenv shell`
+```
+
+You may need to make the script executable. While in the same directory as the script is located, run:
+
+```
+$ chmod +x lat_test.sh
+``` 
+
+Now execute it but running:
+
+```
+./lat_test.sh
+```
+
+What is the output? Don't be too surprised if it's several whole seconds. Unlike a ping request which simply tests reachability, this script includes server side time taken.
+
+![lat_test_on_canyon_in_s3_sydney Page](../../../media/codelabs/codelab-03/lat_test_on_canyon_in_s3_sydney.png)
+
+For comparison, try some other URLs in the script. If you use the link for the index file from codelab-02, the time should be on the order of .5 seconds (recall we used us-east-1 which is located in northern Virginia).
+
+#### CloudFront
+
+Lets return to using the [AWS GUI](https://console.aws.amazon.com/console/home?region=us-east-1#), this time navigating to CloudFront, which can be found under *Storage & Content Delivery*.
+
+From the S3 CloudFront, click `Create Distribution`. This will allow us to tell AWS which origin to use for our content and configure settings. There are two types of delivery CloudFront can use, *web* and *RTMP*.
+
+Click the top `Get Started` button under the _Web_ heading. This is for static (like our canyon image) and dynamic content. _RTMP_ is best for streaming.
+
+Now, click into the *Origin Domain Name* text box. You will see a list of possible origins for content. Select the bucket created in the first part of this tutorial. We'll be using the default values for all other fields in this tutorial.
+
+Scroll to the bottom and click the blue `Create Distribution` in the bottom-left. You will then be on a page showing your distributions that will look something like this.
+
+![dist_status_in_progress Page](../../../media/codelabs/codelab-03/dist_status_in_progress.png)
+
+Note that the status is *in progress*, deployment may take several minutes. When complete, the status will change to *deployed*. Then, go ahead and grab the _Domain Name_ (it should end in .cloudfront.net).
```

Comment:
nit: backticks for URLs, button names, file names, inline scripts, anything else generally "code-like" f.e. AWS regions